tests/org.jboss.tools.batch.ui.bot.test/src/org/jboss/tools/batch/ui/bot/test/editor/jobxml/ValidateSourceRefAttributeTest.java

```diff
@@ -104,6 +105,7 @@ public void testPartitionElementsReference() {
 		referenceCheck(ANALYZER_REF);		
 	}
 	
+	@Ignore("Jira issue: JBIDE-23656")
```

Comment:
This is not a correct handling - you will just forget that this test exists :) . You should use expected exception. Create your own exception, then wrap the appropriate test line with try catch and throw your exception.
Once the issue is fixed the test will start to fail (no expected exception will be thrown)